src/languages/en.ts

```diff
@@ -613,8 +615,22 @@ export default {
         profileAvatar: 'Profile avatar',
     },
     loungeAccessPage: {
+        checkIn: 'Check in',
+        youAreCheckedIn: 'You’re checked in!',
+        addressLabel: 'Address',
+        address: '88 Kearny Street, Floor 16, San Francisco, CA',
+        nextCheckInLabel: 'Next check-in',
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        nextCheckInBeforeNumberCheckIn: ({checkInsRemaining}: RemainingCheckInsParams) => 'You’re eligible to check in to the lounge and have',
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need to add this here since these variables are used?

Reply:
They are not used, at least not in English translation. Those variables are only used in Spanish translation. In English translation they are only declared so that typecheck doesn't find inconsistencies between Spanish and English types:
<img width="1329" alt="image" src="https://github.com/Expensify/App/assets/85628901/6b66ec7c-7e8c-4467-a751-eb0e6fef566a">

Reply:
Maybe I am having a dense day, but I still can't see where this is only used for the Spanish translation and why they are not used for the English translation